services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/select_activities_container.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import AssigningIndicator from '../../shared/button_loading_indicator';
+import AssigningIndicator from '../../shared/button_loading_indicator.tsx';
```

Comment:
Why add the `.tsx` here (and elsewhere)?

Reply:
line 4 had '../assignment_flow_navigation.tsx'; so I was just copying that pattern.  I will remove both since they don't seem to be needed.